modules/SDL.lua

```diff
@@ -1,15 +1,67 @@
 require('os')
 local sdl_logger = require('sdl_logger')
 local config = require('config')
+local console = require('console')
 local SDL = { }
 
 require('atf.util')
 
+SDL.buildOptions = {}
 SDL.exitOnCrash = true
 SDL.STOPPED = 0
 SDL.RUNNING = 1
 SDL.CRASH = -1
 
+local usedBuildOptions = {
+  remoteControl =  {
+    sdlBuildParameter = "REMOTE_CONTROL",
+    defaultValue = "ON"
+  },
+  extendedPolicy =  {
+    sdlBuildParameter = "EXTENDED_POLICY",
+    defaultValue = "PROPRIETARY"
+  }
+}
+
+local function readParameterFromCMakeCacheFile(paramName)
```

Comment:
@aderiabin lack of decription

Reply:
@LuxoftAKutsan  Description in LDoc format was added (see 5aec3b3)